### PR TITLE
Perform Audit for Vulnerabilities

### LINKED ISSUE #442: Perform Audit for Vulnerabilities
# Feature Request

## Description
Perform Audit for Vulnerabilities. When running npm install I am seeing:

screen shot 2019-01-03 at 4 08 32 pm

## Mentions
@mjhenkes @ryanthemanuel @dkasper-was-taken @tbiethman @benbcai

### PR DESCRIPTION
**Pull request by gabeparra01**
### Summary
7 vulnerabilities are reported by npm when running npm install. 6 of these were fixed by updating nightwatch. Fixes #442

### Additional Details
There is still one moderate vulnerability remaining:
image

underscore.string created a fix but argparse did not release a patch for this fix. Instead, argparse just removed underscore.string as a dependency in a major version update. A pull request has been made to remarkable to update their argparse version to the one with the fix:
https://github.com/jonschlinkert/remarkable/pull/312
The repository owner of argpase said that they will eventually merge the pull request but it is low priority because users would have to "attack themselves" to expose the vulnerability.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "lerna": "^2.11.0",
     "link-parent-bin": "^0.2.0",
     "markdown-magic": "^0.1.25",
-    "nightwatch": "^0.9.19",
+    "nightwatch": "^1.0.19",
```
Comment (emilyrohrbough) [inline package.json]: can we remove this dependencies? I know its peer of terra-toolkit, however, we are not doing nightwatch testing in this repo.

Comment (gabeparra01) [reply package.json]: That works too: https://github.com/cerner/terra-clinical/pull/462/commits/6daea92574939c135969b1f0ccacf693608c6a33

**Comment by emilyrohrbough** [comment]
You reported this fixes 6 of 7. What was issue no. 7?

**Comment by gabeparra01** [comment]
@emilyrohrbough underscore.string

**Comment by emilyrohrbough** [comment]
🤦‍♀️ you did mention that...

**Comment by gabeparra01** [comment]
Following up to see if there are any other changes required for this merge

**Pull request merged**